### DISCUSSION TITLE
Is it possible to turn off the manifest validation?

### DISCUSSION THREAD

**Discussion by Blackbaud-JasonBodnar** [aklinker1/vite-plugin-web-extension#146] [Q&A]
My company uses Zscaler for security related things and co-workers who have it installed cannot build my extension. It seems there's something with the SSL cert and Zscaler when this project gets the JSON schema for the manifest validation. There's nothing wrong with this project. I think it has something to do with Zscaler intercepting traffic and using its own SSL cert so we'd need to add the Zscaler cert to whatever keystore vite is using to grab the JSON schema, but it's probably just easier if we turn off the manifest validation.

![image](https://github.com/aklinker1/vite-plugin-web-extension/assets/5076601/4b7bc8d5-90da-40c6-9766-2769467ba026)

**Comment by aklinker1** [comment]
Yes, set `skipManifestValidation: true`.

https://vite-plugin-web-extension.aklinker1.io/config/plugin-options.html#skipmanifestvalidation
